Add documentation for the fritz `dial` action (#40590)
home-assistant/home-assistant.github.io: source/_integrations/fritz.markdown

{% important %}
Both the TR-064 (_Permit access for apps_) and UPnP (_Transmit status information over UPnP_) protocol needs to be enabled in the FRITZ!Box under **Home Network** > **Network** > **Network settings** > **Access Settings in the Home Network** for Home Assistant to login and read device info.

> To use the [ring](#action-ring) action, the click to dial service of the FRITZ!Box must also be enabled under **Telephony** > **Calls** > **Click to Dial**.

Reviewer: ```suggestion
To use the [dial](#action-ring) action, the click to dial service of the FRITZ!Box must also be enabled under **Telephony** > **Calls** > **Click to Dial**.
```

> To use the [dial](#action-ring) action, the click to dial service of the FRITZ!Box must also be enabled under **Telephony** > **Calls** > **Click to Dial**.

Reviewer: ```suggestion
To use the [dial](#action-dial) action, the click to dial service of the FRITZ!Box must also be enabled under **Telephony** > **Calls** > **Click to Dial**.
```

> To use the [`ring` action](#action-ring), in addition to the above the dial-help service of the FRITZ!Box must also be activated under **Telephony** > **Calls** > **Dial help**.

Reviewer: ```suggestion
To use the [ring](#action-ring) action, the click to dial service of the FRITZ!Box must also be enabled under **Telephony** > **Calls** > **Click to Dial**.
```

it seems to be called ["click to dial"](https://fritzhelp.avm.de/help/en/FRITZ-Box-7530-AX/avm/024p2/hilfe_fon_waehlhilfe) (_can confirm that with my 7530ax on FRITZ!OS 8.20_)

## Actions

Available {% term actions %}:

- `set_guest_wifi_password`

> - `ring`

Reviewer: ```suggestion
- `dial`
```

| `password` | no | New password for the guest wifi (_will be auto-generated if not defined_) |
| `length` | no | Length of the auto-generated password. (_default 12_) |

### Action `ring`

> Makes the FRITZ!Box dial a phone number. The dial-help service of the FRITZ!Box must be activated to make this work.

Reviewer: ```suggestion
Makes the FRITZ!Box dial a phone number.
```

This should be mentioned under the prerequisites

Author: It's not a prerequisit for the entire integration, but I will move it up there and make it cleat that it is only a prerequisite for this action.

| `password` | no | New password for the guest wifi (_will be auto-generated if not defined_) |
| `length` | no | Length of the auto-generated password. (_default 12_) |

> ### Action `ring`

Reviewer: ```suggestion
### Action `dial`
```
